bundle/config/experimental.go

```diff
@@ -23,6 +23,27 @@ type Experimental struct {
 	// be removed in the future once we have a proper workaround like allowing IS_OWNER
 	// as a top-level permission in the DAB.
 	UseLegacyRunAs bool `json:"use_legacy_run_as,omitempty"`
+
+	// PyDABs determines whether to load the 'databricks-pydabs' package.
+	//
+	// PyDABs allows to define bundle configuration using Python. For now, this option has to
+	// be used together with 'venv' configuration.
+	PyDABs PyDABs `json:"pydabs,omitempty"`
+
+	// VEnv is the virtual environment configuration for the bundle.
+	//
+	// When enabled, the bundle will use Python interpreter from the virtual environment.
+	VEnv VEnv `json:"venv,omitempty"`
```

Comment:
This PR still uses it only for PyDABs, so I think it's good to move it there until we have a change that makes everything use the venv interpreter (also for artifact building).